components/error-boundary.tsx

```
@@ -1,36 +1,149 @@
+              {error.stack && (
+                <pre className="mt-2 whitespace-pre-wrap">{error.stack}</pre>
+              )}
+            </div>
+          </details>
+        )}
+
+        <div className="flex gap-3 justify-center">
+          <button
+            onClick={resetError}
+            className="inline-flex items-center gap-2 px-4 py-2 rounded-md bg-primary text-primary-foreground hover:bg-primary/90 transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)